src/qdisp/UberJob.h

```diff
@@ -24,9 +24,8 @@
 // System headers
 
 // Qserv headers
-#include "qmeta/types.h"
-#include "czar/CzarChunkMap.h"  // Need nested class. TODO:UJ Make non-nested?
-#include "czar/CzarRegistry.h"  // Need nested class. TODO:UJ Make non-nested?
+#include "czar/CzarChunkMap.h"  // Need nested class.
+#include "czar/CzarRegistry.h"  // Need nested class.
```

Comment:
What does this mean: "Need nested class"? And why is this comment posted here instead of being put into the class description section of the corresponding classes?